packages/Search/test/tstDetailsBatchedQueries.cpp

```diff
@@ -0,0 +1,73 @@
+/****************************************************************************
+ * Copyright (c) 2012-2018 by the DataTransferKit authors                   *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the DataTransferKit library. DataTransferKit is     *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <DTK_DetailsBatchedQueries.hpp>
+
+#include <Teuchos_UnitTestHarness.hpp>
+
+template <typename DeviceType, typename ValueType>
+Kokkos::View<ValueType *, DeviceType> toView( std::vector<ValueType> const &v )
+{
+    Kokkos::View<ValueType *, DeviceType> w( "whocares", v.size() );
+    auto w_host = Kokkos::create_mirror_view( w );
+    for ( int i = 0; i < w.extent_int( 0 ); ++i )
+        w_host( i ) = v[i];
+    Kokkos::deep_copy( w, w_host );
+    return w;
+}
+
+TEUCHOS_UNIT_TEST_TEMPLATE_1_DECL( BatchedQueries, permute_offset_and_indices,
+                                   DeviceType )
+{
+    Kokkos::View<int *, DeviceType> offset( "offset" );
+    Kokkos::View<int *, DeviceType> indices( "indices" );
+
+    Kokkos::View<int *, DeviceType> permute( "permute" );
+
+    TEST_THROW( DataTransferKit::Details::BatchedQueries<
+                    DeviceType>::reversePermutation( permute, offset, indices ),
+                DataTransferKit::DataTransferKitException );
+
+    Kokkos::resize( offset, 1 );
+    TEST_NOTHROW( DataTransferKit::Details::BatchedQueries<
+                  DeviceType>::reversePermutation( permute, offset, indices ) );
+
+    DataTransferKit::Details::BatchedQueries<DeviceType>::permuteOffset(
```

Comment:
Is this a test? If yes either comment or use `TEST_NOTHROW`

Reply:
I think I just forgot to remove that line.